src/App.tsx

```diff
@@ -31,7 +32,7 @@ function App() {
         },
         {
           path: '/home/profile',
-          element: <div>Profile component</div>,
+          element: <ProfilePage profileImage={undefined} />,
```

Comment:
we should consider fetching data directly in profile page in future instead of passing them in props